README.md

```diff
@@ -46,6 +46,7 @@ grafana-cli plugins install volkovlabs-echarts-panel
 - Supports SVG and Canvas renderers.
 - Includes USA and World GeoJSON maps. Additional maps can be loaded dynamically.
 - Includes [Liquid Fill Chart](https://github.com/ecomfe/echarts-liquidfill), which is usually used to represent data in percentage.
+- Includes [WordCloud Chart](https://github.com/ecomfe/echarts-wordcloud), which is usually used to represent data in percentage.
```

Comment:
fix the description

Reply:
Thank you. I will update and tests are failing. I need to fix them as well.